dataplane/standalone/port.h

```diff
@@ -28,13 +32,18 @@ extern "C" {
 
 class Port : public APIBase {
  public:
-  Port(std::shared_ptr<AttributeManager> mgr,
-       std::shared_ptr<forwarding::Forwarding::Stub> c)
-      : APIBase(mgr, c) {}
+  Port(std::string id, std::shared_ptr<AttributeManager> mgr,
+       std::shared_ptr<forwarding::Forwarding::Stub> fwd,
+       std::shared_ptr<lemming::dataplane::Dataplane::Stub> dplane)
+      : APIBase(id, mgr, fwd, dplane) {}
   ~Port() = default;
   sai_status_t create(_In_ uint32_t attr_count,
                       _In_ const sai_attribute_t* attr_list);
   sai_status_t set_attribute(_In_ const sai_attribute_t* attr);
+
+ private:
+  static std::unordered_map<std::string, std::vector<int>> parseLaneMap();
+  static std::unordered_map<std::string, std::vector<int>> laneMap;
```

Comment:
(no action required) What's a lane map?

Reply:
added comment, ports are creates with a list of lanes. the lanemap is map of lanes to veths